doc/output-details-and-examples/src/output-files/eplusout-eio.tex

```diff
@@ -2346,7 +2346,7 @@ \subsubsection{Description of the Windows Construction Report}\label{description
 
 \subsubsection{Fields in Part 1 -- Window Construction Report}\label{fields-in-part-1-window-construction-report}
 
-\textbf{Line 1:} ! \textless{}WindowConstruction\textgreater{},Construction Name,\#Layers,Roughness,Conductance \{W/m2-K\},SHGC,SolarTransmittanceNormalIncid,VisibleTransmittanceNormalIncid
+\textbf{Line 1:} ! \textless{}WindowConstruction\textgreater{},Construction Name,\#Layers,Roughness,Conductance \{W/m2-K\},Conductance (Before Adjusted) \{W/m2-K\},Convection Coefficient Adjustment Ratio,SHGC,SolarTransmittanceNormalIncid,VisibleTransmittanceNormalIncid
```

Comment:
The section below here include a description of each "field". Please add descriptions for the 2 new fields.

Reply:
Also, since you're working in this file already, please update the old names "Material:WindowGlass" etc. to "WindowMaterial:Glazing" etc.

Reply:
And it would be nice to use `\begin{lstlisting}` style for the sample eio lines (e.g. Chiller Standard Rating Information).

Reply:
@mjwitte I added the field descriptions and modified the list to `\begin{lstlisting}`. Can you please take a look if that's what you mean?

Reply:
For the old "Material:WindowGlass" parts, I'm not following. Does it mean the [Material:WindowGlass](https://bigladdersoftware.com/epx/docs/9-6/output-details-and-examples/eplusout-eio.html#materialwindowglass) section should be deleted as [Window Constructions](https://bigladdersoftware.com/epx/docs/9-6/output-details-and-examples/eplusout-eio.html#window-constructions) section is a replacement?

Besides, [part 2 fields ](https://bigladdersoftware.com/epx/docs/9-6/output-details-and-examples/eplusout-eio.html#fields-in-part-2-window-construction-report)in Window Constructions seem to be duplicated with [part 1.](https://bigladdersoftware.com/epx/docs/9-6/output-details-and-examples/eplusout-eio.html#fields-in-part-1-window-construction-report)

Reply:
No, part 2 is for the material layers which follow for each WindowConstruction line. All I was asking for here was to substitute 3 instances of "Material:WindowGlass" (which is from long long ago) to "WindowMaterial:Glazing". And likewise for Material:WindowGas, Material:WindowShade, Material:WindowBlind, and Material:WindowScreen.

Reply:
Hope the latest commit fixed it.